templates/main.html

```diff
@@ -21,6 +21,17 @@
       </a>
       <h1 data-i18n="page-title">Google Code-in 2017 Current Leaders</h1>
     </div>
+    <div class="ribbon">
+      <a href="https://your-url" class="github-corner" aria-label="View source on Github">
+        <svg width="80" height="80" viewBox="0 0 250 250" style="fill:#151513; color:#fff; position: absolute; top: 0; border: 0; right: 0;" aria-hidden="true">
```

Comment:
Nobody likes *that* guy, but this code is licensed under MIT. MIT is a subset of (A)GPL, so it's fine importing that code as long as we don't change the copyright. Maybe just mention that this code is from https://github.com/tholman/github-corners/blob/master/license.md, which is MIT licensed and preserve the copyright notice?

Not sure what's the right approach here. Maybe I'm just being unnecessarily pedantic. CC @jayvdb